### ISSUE TITLE
Blast Corps - EU

### ISSUE THREAD

**Issue by Invictaz** [Rinnegatamante/DaedalusX64-vitaGL-Compatibility#38]
# Game summary
- Game name: Blast Corps
- Game version: EU

# Daedalus X64 summary
- Version: v0.1

# Issues
Shows intial logo's then nothing. Crashes on black screen with white controller.
Even with interpreter mode.

**Comment by Invictaz** [comment]
Same in 0.3

**Comment by Rinnegatamante** [comment]
On latest nightly it now works perfectly.

**Comment by jameslstill** [comment]
When sound effects / speech play I get slowdown.  Still playable but not smooth.  Have tested with lolicon500 and get same result.

**Comment by JosepMC154** [comment]
It's now fullspeed with audio.

**Comment by jameslstill** [comment]
That is amazing!  Plays great.  The icons of the vehicles on the stage select screen are a bit messed up - but don't affect playability at all.

**Comment by Invictaz** [comment]
@Rinnegatamante It's not "perfect". It has serious speedup issues in 0.6 in the second level where the audio drops frequently. Renderer needs to be at legacy and the demo mode (where "press start" is shown) randomly cuts out the audio. In 500mhz overclock mode the audio changes pitch a lot.

**Comment by rufotheone-fr** [comment]
[Blast Dozer](https://github.com/Rinnegatamante/DaedalusX64-vitaGL-Compatibility/issues/268) in NTSC-J

**Comment by Damderiam** [comment]
Collision seems very messed up in 0.6.  Don't know about earlier versions.

In the first level, almost any interaction with the wooden platform on the train tracks near where you start will result in you getting stuck inside of it.  Additionally, the truck seems to be exploding upon contact with minor objects (fences, signs, etc.) which it's not supposed to do.  It doesn't seem to do it until after a message has been displayed.

Tried toggling every option in the Emulation menu to no effect.  Game won't run unless you're using DynaRec.

Also, trying to use sync audio will lock the game up, but that's not really a big deal.

**Comment by Invictaz** [comment]
Exactly, the game feels off when you are colliding into some elements.

**Comment by adrianbass360** [comment]
US VERSION PLAYABLE WITHOUT SOUND. For the game to run decently without any of the glitches mentioned above, disable sound and use daedalusx64 v0.5 with cached on. You can use this config and this older daedalusx64 version

[Configs.zip](https://github.com/user-attachments/files/19817395/Configs.zip)
[DaedalusX64 v0.5 (STABLE).zip](https://github.com/user-attachments/files/19817396/DaedalusX64.v0.5.STABLE.zip)
